socorro/signature/tests/test_rules.py

```diff
@@ -321,23 +321,6 @@ def test_generate_4(self):
         sig, notes = s.generate(a)
         assert sig == 'f | e | d | i'
 
-    def test_specifiers_and_return_types(self):
-        """prefix and irrelevant should match ignoring specifiers and return types"""
-        generator = self.setup_config_c_sig_tool(
-            # irrelevant
-            ig=['ignored'],
-            # prefix
-            pr=['pre1', 'pre2'],
-        )
-        source_list = (
-            'static void ignored',
-            'pre1',
-            'static bool pre2',
-            'foo'
-        )
-        sig, notes = generator.generate(source_list)
-        assert sig == 'pre1 | static bool pre2 | foo'
```

Comment:
`generate` runs on normalized function signatures, so this test isn't applicable anymore.